src/libraries/System.Transactions.Local/src/System/Transactions/DtcProxyShim/NotificationShimFactory.cs

```diff
@@ -42,6 +42,7 @@ private void Initialize(SafeWaitHandle hEvent)
         // TODO: Instantiate all the locks (critical sections)
     }
 
+    [System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessage("Trimming", "IL2050", Justification = "Leave me alone")]
```

Comment:
While I understand the frustration... maybe we could come up with a slightly better justification 😉

Reply:
Indeed. My first round had a slightly more colorful justification. This PR is meant as merely a way to move forward and does need to be updated.

Reply:
Oh - you had even better one? Now I'm curious :smile:
Anyway - it was just a jab, I'm sure we would never put this into the real source base.